test/org/sagebionetworks/bridge/dynamodb/DynamoAppConfigTest.java

```diff
@@ -56,6 +81,7 @@ public void canSerialize() throws Exception {
         criteria.setMaxAppVersion(ANDROID, 15);
         criteria.setLanguage("fr");
 
+        String clientDataString = TestUtils.createJson("{'booleanFlag':true,'stringValue':'testString','intValue':4}");
         JsonNode clientData = TestUtils.getClientData();
```

Comment:
We create clientDataString from hardcoded JSON, but we use TestUtils.getClientData(). If we ever change TestUtils.getClientData() for whatever reason, this test breaks.